### PR TITLE
Optimize/round decimal number

### LINKED ISSUE #147: [optimize] round decimal number by 2 digit
Image

### PR DESCRIPTION
**Pull request by mhmd-android**
**Explanation of Changes in Pull Request**

1. **Enhanced Number Formatting**

- I've updated the 'MarketListItem' to format the `priceChangePercentage24h` with two decimal places for a consistent and clear display of percentage changes. This change ensures that our users can easily interpret and compare these values.

2. **Test Coverage**

- The corresponding test suite, `RoundToTwoDecimalPlacesTest`, has been updated to verify the correctness of the formatting logic, covering various input scenarios.

> The goal of this PR is to provide a more polished and consistent user interface, which can positively impact user satisfaction. It also ensures that our codebase remains robust and test-covered.

**shoot:**

Shot 0001

Thank you for your time and collaboration

Best regards.

Close #147

### CODE REVIEW

File: library/designsystem/src/main/java/ir/composenews/designsystem/extensionFunction/Number.kt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package ir.composenews.designsystem.extensionFunction
+
+fun Number.roundToTwoDecimalPlaces(): String {
```
Comment (Kaaveh) [inline library/designsystem/src/main/java/ir/composenews/designsystem/extensionFunction/Number.kt:3]: I propose to move this function into a new module named `:core:extensions`

File: library/designsystem/src/test/java/ir/composenews/designsystem/RoundToTwoDecimalPlacesTest.kt [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package ir.composenews.designsystem
+
+import ir.composenews.designsystem.extensionFunction.roundToTwoDecimalPlaces
+import org.junit.Assert
+import org.junit.Test
+import java.math.BigDecimal
+
+class RoundToTwoDecimalPlacesTest {
```
Comment (Kaaveh) [inline library/designsystem/src/test/java/ir/composenews/designsystem/RoundToTwoDecimalPlacesTest.kt:8]: It's amazing you wrote test for it! 😍

**Review by Kaaveh — CHANGES_REQUESTED** [review]

File: library/designsystem/src/main/java/ir/composenews/designsystem/extensionFunction/Number.kt (hunk 1 above)
Comment (mhmd-android) [reply library/designsystem/src/main/java/ir/composenews/designsystem/extensionFunction/Number.kt:3]: yes, that's a perfect idea, I agree with you, and I will move this function and its test into an extension module

**Review by Kaaveh — APPROVED** [review]

**Comment by Kaaveh** [comment]
Thanks, @mhmd-android ! 🙌🏻

**Pull request merged**